### DISCUSSION TITLE
I cannot run: "multi-segment language detection" and "Batched faster-whisper"

### DISCUSSION THREAD

**Discussion by toanhn** [SYSTRAN/faster-whisper#981] [Q&A]
I cannot run "multi-segment language detection" and "Batched faster-whisper":
- https://github.com/SYSTRAN/faster-whisper?tab=readme-ov-file#multi-segment-language-detection
- https://github.com/SYSTRAN/faster-whisper?tab=readme-ov-file#batched-faster-whisper

Can you please help me to use the above features, thank you so much.

**Comment by toanhn** [comment]
![Selection_269](https://github.com/user-attachments/assets/1e99f64c-9452-46f2-ad6a-9f6bda437209)
![Selection_270](https://github.com/user-attachments/assets/6f2829da-12a5-4ff8-a322-2c9d5eb71684)

**Comment by MahmoudAshraf97** [comment]
you need to install from source, these features are not available in pip package yet

**Reply by toanhuynhnguyen** [reply]
Thank you so much.
